flixel/tile/FlxTilemap.hx

```diff
@@ -115,6 +115,8 @@ class FlxTilemap extends FlxBaseTilemap<FlxTile>
 	private var _scaledTileHeight:Float = 0;
 	
 	#if FLX_DEBUG
+	public var debugHideDecorativeTiles:Bool = true;
```

Comment:
I don't like the idea of introducing new terminology for this (what is considered a "decorative" tile? seems a bit ambigious). In general, I would prefer a more flexible solution (setting the color per debug tile type like you originally proposed sounds good, that way you'd have full control, similar to `debugBoundingBoxColor` in `FlxObject`. Which makes me wonder what that does for `FlxTilemap`, if anything...).